s4e-backend/src/main/java/pl/cyfronet/s4e/admin/product/AdminProductController.java

```diff
@@ -25,6 +28,70 @@
 public class AdminProductController {
     private final ProductService productService;
     private final AdminProductMapper adminProductMapper;
+    private final ProductCategoryRepository productCategoryRepository;
+
+    @Operation(summary = "Create Product Categories")
```

Comment:
I'd rather move this operation in a separate package `s4e.admin.category` as categories are separate entities.